working_villagers/jobs/follow_player.lua

```diff
@@ -47,11 +47,22 @@ function follower.step(v)
   end
 end
 
+local follower_tools = {
+	-- need a 3d armor light for the villagers
+	["default:torch"] = 99,
+}
+-- TODO collect nearby items that the player has dropped, etc. (e.g., by digging)
```

Comment:
This was more of a testing and logistics job. Maybe a separate job could be created or the torcher could be modified for more "active" following/accompanying.

Reply:
The follow player logic is a good starting point for bots that need more real-time pathfinding, such as a thief, who follows the nearest player whose inventory contains anything good. It will likely form some of the core combat logic. It also makes the mayor way more useful.

Maybe we can include settable options or different modes as with the guard.